contracts/exchange/contracts/src/MixinSignatureValidator.sol

```diff
@@ -244,7 +244,17 @@ contract MixinSignatureValidator is
             hash,
             signature
         );
+        bytes32 magicValue = bytes32(bytes4(keccak256("isValidWalletSignature(bytes32,address,bytes)")));
         assembly {
+            if iszero(extcodesize(walletAddress)) {
```

Comment:
Let's add a comment that the `extcodesize` checks are redundant, but were added as an extra safety measure.